makefiles/vars.inc.mk

```diff
@@ -49,6 +49,7 @@ export OBJDUMPFLAGS          # The parameter for OBJDUMP.
 export SIZE                  # The command to read to size of the ELF sections.
 export UNDEF                 # Set by the BOARD's and CPU's Makefile.include, this contains object files with must not be used in the ELFFILE even if the if no call to the functions.
 export WERROR                # Treat all compiler warnings as errors if set to 1 (see -Werror flag in GCC manual)
+export FLASH_OFFSET          # Define an offset to flash code into ROM memory.
```

Comment:
Actually this might be a good place. My suggestion would be, to insert a section for setting default values on top of this file, as in 
```make
# set default values for selected global variables
FLASH_ADDR ?= 0

# export global variables defined by the make system
export Q                     # Used in front of Makefile lines to suppress the printing of the command if user did not opt-in to see them.
export QQ                    # as Q, but be more quiet
...
```

Further I would prefer to (i) name the varialbe `FLASH_ADDR` and second to put the export line it it's own block (as this define is not tool related but CPU related)